.travis.yml

```diff
@@ -1,7 +1,13 @@
+dist: trusty
```

Comment:
The reminder of this change was "CPC" being flagged by spell-check in Travis but not locally; I suspected that travis's default of *precise* might be to blame.

It wasn't, but we would still prefer to run on trusty rather than precise.